tools/update-webpage.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright 2015-2016 Samsung Electronics Co., Ltd.
+# Copyright 2016 University of Szeged
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ $# -ne 1 ]; then
+  echo "Please, specify your gh-pages clone directory: update-webpage <gh-pages clone dir>"
+  exit 1
+fi
+
+gh_pages_dir=$1
+docs_dir="../docs"
+
+GETTING_STARTED_MD="GETTING-STARTED.md"
+API_REFERENCE_MD="API-REFERENCE.md"
+API_EXAMPLES_MD="API-EXAMPLE.md"
+INTERNALS_MD="INTERNALS.md"
+
+declare -A titles
+declare -A permalink
+declare -A order
+
+titles[$GETTING_STARTED_MD]="Getting Started"
+titles[$API_REFERENCE_MD]="API Reference"
+titles[$API_EXAMPLES_MD]="API Examples"
+titles[$INTERNALS_MD]="Internals"
+
+permalink[$GETTING_STARTED_MD]='/getting-started/'
+permalink[$API_REFERENCE_MD]='/api-reference/'
+permalink[$API_EXAMPLES_MD]='/api-example/'
+permalink[$INTERNALS_MD]='/internals/'
+
+order[$GETTING_STARTED_MD]='01.'
+order[$API_REFERENCE_MD]='02.'
+order[$API_EXAMPLES_MD]='03.'
+order[$INTERNALS_MD]='04.'
```

Comment:
The ordering determines in which order the menus will appear on the webpage (https://ktorpi.github.io/jerryscript/) and I think it would be nice to have e.g. the Getting Started page before the API reference.